system-tests/projects/config-cjs-and-esm/config-with-ts-module/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
-    "find-up": "6.3.0"
+    "find-up": "6.3.0",
+    "typescript": "4.7.3"
```

Comment:
A lot of these system tests are very misleading - you cannot have a typescript without `typescript` in the `dependencies`, for example.